.github/CONTRIBUTING.md

```diff
@@ -150,18 +159,73 @@ about how it works.
 
 ### Documentation
 
-The main documentation is written in [Markdown][] using [MkDocs][]. Dredd uses
-[ReadTheDocs][] to build and publish the documentation:
+Dredd's documentation is written in [Markdown][] using [Sphinx][]. [ReadTheDocs][] is used to build and publish the documentation:
 
 - [https://dredd.readthedocs.io](https://dredd.readthedocs.io) - preferred long URL
 - [http://dredd.rtfd.org](http://dredd.rtfd.org) - preferred short URL
 
-Source of the documentation can be found in the [docs][] directory. To contribute to Dredd's documentation, you will need to follow the [MkDocs installation instructions](http://www.mkdocs.org/#installation). Once installed, you may use following commands:
+Source of the documentation can be found in the [docs][] directory. To render Dredd's documentation on your computer, you need Python 3 and Node.js installed.
+
+#### Installation and Development
+
+1.  Make sure `node` is an executable and `npm install` has been done for the Dredd directory. Extensions to the docs are written in Node.js and Sphinx needs to have a way to execute them.
+2.  [Get Python 3](https://www.python.org/downloads/). On macOS, run `brew install python3`.
+3.  Create a [virtual environment](https://docs.python.org/3/library/venv.html) and activate it:
+
+    ```sh
+    python3 -m venv ./env
```

Comment:
I think it's more common to call it `venv` over `env`. Prevents any confusion with environment variables files (often in `.env` files).